packages/plugin-core/src/commands/VaultAddCommand.ts

```diff
@@ -169,15 +169,13 @@ export class VaultAddCommand extends BasicCommand<CommandOpts, CommandOutput> {
           await wsService.addWorkspace({ workspace });
           await this.addWorkspaceToWorkspace(workspace);
         } else {
-          await _.reduce(
-            vaults,
-            async (resp: any, vault: DVault) => {
-              await resp;
-              await wsService.createVault({ vault });
-              return this.addVaultToWorkspace(vault);
-            },
-            Promise.resolve()
-          );
+          // Some things, like updating config, can't be parallelized so needs to be done one at a time
+          for (const vault of vaults) {
```

Comment:
TIL - didn't realize that `for` loops can be used to run things in serial!

might be worth adding as a utility in common-all since we do this in a number of places